packages/react/src/lib/felaDisableAnimationsPlugin.ts

```diff
@@ -0,0 +1,34 @@
+import * as _ from 'lodash'
+
+/**
+ * Fela plugin for disabling animations. The animations are disabled or not based on the
+ * props' disableAnimations param. If the value of the prop is true, all animation related
+ * styles are removed.
+ *
+ * Caution! Infinite recursion is possible in case if style object has links to self in the props
+ * tree.
+ */
+export default () => {
+  const disableAnimations = (styles: Object, type?, renderer?, props?) => {
+    if (props && props.disableAnimations && type && type === 'RULE') {
```

Comment:
`type && type === 'RULE'` is too strict, `type === 'RULE'` is enough

```suggestion
    if (props && props.disableAnimations && type === 'RULE') {
```